src/ape/api/transactions.py

```diff
@@ -30,6 +30,7 @@ class TransactionAPI(BaseInterfaceModel):
     receiver: Optional[str] = Field(None, alias="to")
     sender: Optional[str] = Field(None, alias="from")
     gas_limit: Optional[int] = Field(None, alias="gas")
+    gas_price: Optional[int] = Field(None, alias="gasPrice")
```

Comment:
This is a somewhat significant change here. I think it wasn't originally here because not all transaction dicts have this field, so it was open to subclasses to define extra elements they have and compute some sort of "total cost" property (that I don't see around)